packages/contracts-bedrock/snapshots/semver-lock.json

```diff
@@ -19,9 +19,13 @@
     "initCodeHash": "0x11e28569436e16691f03820e0fd5252492706f5855b350439f695c7e4cd331c3",
     "sourceCodeHash": "0x11b35ee81f797b30ee834e2ffad52686d2100d7ee139db4299b7d854dba25550"
   },
+  "src/L1/OPCMValidator.sol:OPCMValidator": {
+    "initCodeHash": "0xe2351f57183ad2b6ceb7494c7a03473b9c98f64db6a23794f1ee182e59221278",
+    "sourceCodeHash": "0x9ae95f75f8dca15ff6193dfe7c7a23216101e001b1713c0d1d2e52d3a01db16f"
+  },
```

Comment:
I think we should remove `ISemver` from `OPCMValidator` in order to align with the other OPCM components.